unit-test/fm_cmd_utils_tests.c

```diff
@@ -154,161 +154,142 @@ void Test_FM_VerifyOverwrite_OverwriteIsFalse(void)
 }
 
 /* **************************
- * LoadOpenFileData
+ * GetOpenFilesData Tests
  * *************************/
-/*
-void Test_LoadOpenFileData_OpenFileCountUnchanged(void)
+void Test_FM_GetOpenFilesData_OSForEachObjectCalled(void)
 {
     // Arrange
-    uint32 dummy_id = 333;
-    void * dummy_args = NULL;
-
-    uint32 count_openfile_before = OpenFileCount;
+    FM_OpenFilesEntry_t *dummy_files_data = NULL;
 
     // Act
-    LoadOpenFileData(dummy_id, dummy_args);
-
-    uint32 count_openfile_after = OpenFileCount;
+    uint32 open_file_count = FM_GetOpenFilesData(dummy_files_data);
 
     // Assert
-    UtAssert_True(count_openfile_after == count_openfile_before,
-                    "Openfile value is %u and should be %u",count_openfile_after,count_openfile_before);
+    UtAssert_True(open_file_count == 0, "OpenFileCount is %u and should be 0", open_file_count);
 }
 
-void Test_LoadOpenFileData_OpenFileCountIncrease(void)
+void Test_FM_GetOpenFilesData_IncrementOpenFileCount(void)
 {
     // Arrange
-    uint32 dummy_id = 333;
-    void * dummy_args = NULL;
+    osal_id_t id = OS_OBJECT_ID_UNDEFINED;
+    FM_OpenFilesEntry_t dummy_files_data[1];
+    char InitialName[OS_MAX_API_NAME] = "OpenFilesData_test";
 
-    UT_SetDefaultReturnValue(UT_KEY(OS_IdentifyObject),OS_OBJECT_TYPE_OS_STREAM);
-    uint32 count_openfile_before = OpenFileCount;
+    strncpy(dummy_files_data[0].LogicalName, InitialName, OS_MAX_API_NAME);
+    strncpy(dummy_files_data[0].AppName, InitialName, OS_MAX_API_NAME);
 
-    // Act
-    LoadOpenFileData(dummy_id, dummy_args);
+    OS_OpenCreate(&id, NULL, 0, 0);
+    UT_SetDataBuffer(UT_KEY(OS_ForEachObject), &id, sizeof(id), false);
 
-    uint32 count_openfile_after = OpenFileCount;
+    // Act
+    uint32 open_file_count = FM_GetOpenFilesData(dummy_files_data);
```

Comment:
Combine this with line 198... just assert the return

Reply:
`UtAssert_UINT32_EQ(FM_GetOpenFilesData(dummy_files_data), 1);`

Remove open_file_count declaration and the test below.  Same pattern for the rest of the tests.